code/src/index.css

```diff
@@ -1,13 +1,133 @@
+* {
+  margin: 0; 
+  padding: 0;  
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: content-box;
+  -webkit-font-smoothing: antialiased; 
+  text-rendering: optimizeLegibility;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
+  font-family: -apple-system, BlinkMacSystemFont, Helvetica;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #181818;
+  color: #fff;
+}
+
+.container {
+  margin: 2rem;
+}
+
+a {
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 14px;
+}
+
+.artist-name {
+  color: #a0a0a0;
+  font-size: 14px;
+}
+
+.album-name a {
+  font-weight: 700;
+  color: #ffffff;
+}
+
+.album-name {
+  margin: 10px 0;
+}
+
+.album {
+  display: flex;
+  flex-direction: column;
+  max-width: 70vw;
+  padding: 15px;
+}
+
+.flex-container {
+  margin: 2rem auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.artist-container {
+  display: flex;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.image-container {
+  width: 100%;
+  position: relative;  
+}
+
+.icons-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  align-items: center;
+  justify-items: center;
+  justify-content: center;
+  width: 100%
+}
+
+.icons-container a {
+  width: 75%; 
+}
+
+.overlay-container {
+  display: flex;  
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0,0,0,0.6);
+  transition: opacity .3s; 
+  opacity: 0;
+  align-items: center;
+}
+
+.icon {
+  filter: invert(100%) sepia(0%) saturate(0%) hue-rotate(138deg) brightness(103%) contrast(101%);
+  width: 35%;
+}
+
+.image {
+  display: block;
+  width: 100%;  
+}
+
+.play {
+  transition: transform .3s;
+  width: 100%;
+}
+
+@media (min-width: 768px) {
+  .album {
+    max-width: 40vw;
+  }
+}
+
+@media (min-width: 1025px) {
+  a:hover {
+    text-decoration: underline;
+  }
+
+  .overlay-container:hover {
+   opacity: 1; 
+  }
+
+  .play:hover {
+  transform: scale(1.5);
+  }
```

Comment:
Really cool how you guys did this hover effect! That the icon expands without moving the other icons. Very nice detail -  I should incorporate that in my own project.